integration_tests/dbt_project.yml

```diff
@@ -0,0 +1,50 @@
+name: 'integration_tests'
+version: '1.0.0'
+config-version: 2
+
+profile: 'default'
+
+
+
+vars:
+### optional overrides for input data, only for integration_test or use as a project
+#  claims_preprocessing_input_database: tuva               # optional: name of the database where sources feeding this prorject are stored
```

Comment:
Similar comment as in the package dbt_project.yml file. "Claims_preprocessing" is listed a few times. Not sure if that is intentional or should be replaced by "readmissions".